src/routes/+layout.svelte

```diff
@@ -42,8 +44,9 @@
 	}
 
 	onMount(() => {
-		// Check if the user has already decided.
+		new DateTriggerEventHandler(themeEvents);
 
+		// Check if the user has already decided.
```

Comment:
add a newline after this to clean up the diff

Reply:
resolved in 56814c9